# Internal_QA

## To use this program, you can run it from the command line with the following arguments:
{ python sync.py [source folder] [replica folder] [sync interval in seconds] [log file] }

## If code didnt run, maybe your API dont have a permision to yours directory. You can fix it by this code:
{ sudo python sync.py [source folder] [replica folder] [sync interval in seconds] [log file] }
